From a53fc7ec19ab02073b79a8f198e72706e47b9a69 Mon Sep 17 00:00:00 2001
From: Haikel Guemar <hguemar@fedoraproject.org>
Date: Thu, 11 Aug 2016 16:48:28 +0200
Subject: [PATCH] Add suds.reader.DocumentReader compat methods with 0.6
 release

This is required to support oslo.vmware unit tests
---
 suds/reader.py | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/suds/reader.py b/suds/reader.py
index 1454c34..f917d08 100644
--- a/suds/reader.py
+++ b/suds/reader.py
@@ -194,3 +194,10 @@ class DocumentReader(Reader):
         content = ctx.document
         sax = suds.sax.parser.Parser()
         return sax.parse(string=content)
+
+    # compat methods with 0.6
+    def download(self, url):
+        return self.__fetch(self, url)
+
+    def cache(self):
+        return self.__cache(self)
-- 
2.9.0

